README.md

```diff
@@ -161,6 +161,19 @@ Generate a script that points KUBECONFIG to the targeted cluster for the specifi
 eval $(gardenctl kubectl-env bash)
 ```
 
+To load the kubectl configuration for each bash session add the following line at the end of the `~/.bashrc` file:
+
+```bash
+eval "$(gardenctl kubectl-env bash)"
+```
+
+You will need to start a new shell for this setup to take effect.
+Alternatively, you can run the following command to apply it to your current shell session:
+
+```bash
+source ~/.bashrc
+```
+
```

Comment:
we do not make it more complicated / bloat the readme with such optimisations, hence I would vote to omit it.
```suggestion
```

Reply:
Fixed via [`f873fc9` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/f873fc9e63810414eb1e9001feeb7f59708820fa)